_includes/partners/header.html

```diff
@@ -46,31 +46,31 @@
             class="usa-nav__link {% if page.url contains '/our-services' %}usa-current{% endif %}"
             href="{{ '/partners/our-services/' | locale_url }}"
           >
-            Our services
+            <span>Our services</span>
```

Comment:
I removed these in https://github.com/18F/identity-site/pull/842 because I thought they were empty tags that didn't do anything, but it looks like these have meaning. There must be a `.class-name > span` CSS style or something...can we update it to have a class name so we can make it clearer why it's here? Ex `.class-name > .class-name__inner` or something

Reply:
Sure, will make those updates. I agree it is a little confusing.

Reply:
@zachmargolis  updated, thts?

Reply:
yup, thanks!